packages/pwa-kit-react-sdk/src/ssr/browser/main.jsx

```diff
@@ -78,24 +78,27 @@ export const start = () => {
     }
 
     const error = window.__ERROR__
+    const id = uuid()
```

Comment:
It looks like you are implementing the client-side portion that is out of scope. I had a conversation with @johnboxall and we both had the idea to create a "virtual" page view. Meaning for each page navigation we'd generated a new uuid to be used as a correlation id. 

With this extra complexity you might want to choose not to do the client work that is out of scope. 

As an aside, I think implementing these virtual page view will be pretty easy with the way this pr work. The provider just needs a way "reset" the uuid/correlation id. You might want to think about this before finalizing the pr as it might imply there is a change to the provider or hooks depending on what API you come up with.